MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Transform axis value to pixel value and vice-versa | Syncfusion
+description: Learn here all about to transform axis value to pixel value and vice-versa in SfCartesianChart in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Transform axis value to pixel value and vice-versa
+
+[`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1%2Ctabid-3%2Ctabid-6%2Ctabid-8%2Ctabid-10%2Ctabid-12%2Ctabid-25%2Ctabid-27%2Ctabid-29%2Ctabid-23%2Ctabid-14%2Ctabid-37%2Ctabid-17%2Ctabid-19%2Ctabid-21) offers two utility methods to transform the pixel to chart point and vice-versa.
+
+* [`ValueToPoint(double value)`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ValueToPoint_System_Double_) - Converts the data point value to screen point.
```

Comment:
API links were wrong, and code snippet was wrong.

Reply:
I addressed the PR comments
I made the required changes